docs/build_custom_logstash_image.md

```diff
@@ -0,0 +1,62 @@
+# Build A Custom Image with Logstash
+
+If you are looking for a logstash image version with opensearch plugin not available in the [official Docker repository tags](https://hub.docker.com/r/opensearchproject/logstash-oss-with-opensearch-output-plugin/tags) or if you want to include other plugins in your image, it is possible to build a custom image.
+
+## Logstash 8.x
+
+Below is an example of a `Dockerfile` to install `Opensearch 2.0.2` for **`Logstash 8.x`**
```

Comment:
So one has to create this Dockerfile? Say that.

Reply:
Done